awjf_pub/awjf_pub_wap_bst.sql

```diff
@@ -0,0 +1,4 @@
+SELECT ogc_fid,st_multi(wkb_geometry) as wkb_geometry,id_wp,fid_amtei,fid_fk,fid_fr,wirt_zone,gem_bfs,fid_we,gb_flaeche,we_text,fid_eigcod,fid_eig,fid_prod,wpnr,wptyp,betriebsteil,
```

Comment:
Renaming ogc_fid to t_id and wkb_geometry to geometrie
better split the attribute listing earlier (perhaps you need three lines)